app/pages/items/description/ui.test.js

```diff
@@ -105,22 +106,25 @@ test('should render save button', async (t) => {
   await pageSetup(t, true);
   await t.navigateTo(pageUrl);
 
-  const button = Selector('[data-test-id="save-button"] a');
+  const button = Selector('[data-test-id="save-button"] button');
 
   await t
     .expect(button.exists).ok()
-    .expect(await extractInnerText(button)).eql(content.saveButtonText)
-    .expect(button.getAttribute('href')).eql('#');
+    .expect(await extractInnerText(button)).eql(content.saveButtonText);
 });
 
-test('should navigate to ? when save button is clicked', async (t) => {
+test('should navigate to task list page when valid description is added and save is clicked', async (t) => {
+  nock(orderApiUrl)
+    .post('/api/v1/orders')
+    .reply(200, { orderId: 'order1' });
+
   await pageSetup(t, true);
   await t.navigateTo(pageUrl);
 
-  const button = Selector('[data-test-id="save-button"] a');
+  const saveButton = Selector('[data-test-id="save-button"] button');
 
   await t
-    .expect(button.exists).ok()
-    .click(button)
-    .expect(getLocation()).eql('http://localhost:1234/organisation/neworder/description#');
+    .expect(saveButton.exists).ok()
+    .click(saveButton)
+    .expect(getLocation()).eql('http://localhost:1234/order/organisation/order1');
```

Comment:
I've been using the baseUrl for the `/order`. but happy to go with either way. I actually prefer it the way you have done it. Long as we stick to the one way

Reply:
I did have that but then I took it out 🤷 doesn't make any difference as it is the same for all envs